examples/notebooks/vanilla-gan-tutorial.ipynb

```diff
@@ -0,0 +1,1000 @@
+{
```

Comment:
<p>1)you have already imported torch in the previous section</p><p>2)why should you import numpy by 2 different names?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-team/catalyst/pull/660/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>